cli/ops/compiler.rs

```diff
@@ -58,23 +59,25 @@ pub fn op_fetch_source_files(
     futures.push(fut);
   }
 
-  // WARNING: Here we use tokio_util::block_on() which starts a new Tokio
-  // runtime for executing the future. This is so we don't inadvertently run
-  // out of threads in the main runtime.
-  let files = tokio_util::block_on(futures::future::join_all(futures))?;
```

Comment:
This is awesome!
I suspect removing this block_on will dramatically effect the thread count benchmark.

Reply:
This branch:
```
  "thread_count": {
    "error_001": 5, 
    "cold_relative_import": 7, 
    "cold_hello": 5, 
    "workers_round_robin": 3, 
    "relative_import": 3, 
    "workers_startup": 3, 
    "hello": 3
  }, 
```

master branch:
```
  "thread_count": {
    "error_001": 5, 
    "cold_relative_import": 79, 
    "cold_hello": 75, 
    "workers_round_robin": 3, 
    "relative_import": 3, 
    "workers_startup": 3, 
    "hello": 3
  }, 
```